lib/spaceship/tunes/build_train.rb

```diff
@@ -72,13 +72,29 @@ def setup
         end
       end
 
+      # @return (Spaceship::Tunes::Build) The latest build for this train, sorted by upload time.
+      def latest_build
+        @builds.max_by {|build| build.upload_date }
```

Comment:
Space between { and | missing.<br>Pass `&:upload_date` as an argument to `max_by` instead of a block.